diff --git a/build/src/utils/npm.js b/build/src/utils/npm.js
index b861322e982f936bd3267c507a56f48213cc3d16..84ff17ce20f3a13033db6dc6401a41210368cff1 100644
--- a/build/src/utils/npm.js
+++ b/build/src/utils/npm.js
@@ -60,7 +60,7 @@ function _stream() {
     return data;
 }
 function _tar() {
-    const data = /*#__PURE__*/ _interopRequireDefault(require("tar"));
+    const data = { default: require("tar") };
     _tar = function() {
         return data;
     };
diff --git a/build/src/utils/tar.js b/build/src/utils/tar.js
index 8bf12d812646724089f6cd265b16093e8f518570..ce1ba83e71f15341348688139fd476223cb9dde9 100644
--- a/build/src/utils/tar.js
+++ b/build/src/utils/tar.js
@@ -14,7 +14,7 @@ function _spawnAsync() {
     return data;
 }
 function _tar() {
-    const data = /*#__PURE__*/ _interopRequireDefault(require("tar"));
+    const data = { default: require("tar") };
     _tar = function() {
         return data;
     };
